fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/IndexingScrubMissing.java

```diff
@@ -227,7 +234,8 @@ public IndexingScrubMissing(@Nonnull final IndexingCommon common,
                     if (LOGGER.isWarnEnabled() && logWarningCounter > 0) {
                         logWarningCounter --;
                         LOGGER.warn(KeyValueLogMessage.build("Scrubber: missing index entry",
-                                LogMessageKeys.KEY, rec.getPrimaryKey().toString())
+                                        LogMessageKeys.KEY, rec.getPrimaryKey().toString(),
```

Comment:
There's a part of me that would actually like to log the record itself, however (a) that could result in an overly large log message and (b) the log could then leak sensitive PII ... so, we'll keep it at just the PK for now.